### DISCUSSION TITLE
Is it really going to be rewarding as an investment for small guys?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#2827] [Q&A]
Given how fast the netspace is increasing, it seems that the probability of being rewarded is getting diminished for individual farmers. Catching the speed of netspace increase ratio is now almost next to impossible. Given the fixed reward a day for the entire netspace, it is not difficult to see that the little HDD spaces each individual farmer hold is becoming lottery tickets with a constant energy bill, as you have to keep them up 7/24 (HDDs do consume significant energy if you own many + the farming computer that has to run them).  Without pooling, the probability of winning even one CHX would be extremely slim for new individual farmers.  The return is getting diminished quickly vs. the investment on the hardware and electricity bills.

To keep up with the netspace increasing rate, you have to continuously invest in computing power ($30-$40 per Plot/Day rate in today's pricing), plus additional HDD space rate one has to keep up.

Eventually, if we assume all HDDs and spare HDD spaces in the market, plus all new production of HDDs/SSDs were all to enter the Chia race, it might become next to impossible even for smaller pools to gain anything quickly. For BTC, and other coins, there are ASIC miners that would make mining still profitable today for individuals.  With large number of HDDs that has to keep up, the amount of investment to make this profitable as a farmer is becoming questionable.

This whole Chia scheme benefits the early comers, and the return is getting exponentially squeezed day by day... Sigh!

If you are not making a calculated investment to get more definitive returns, it may not be wise to throw in significant savings into Chia. It is like in the early days of certain lottery system, there is a threshold to make it worthwhile. For Chia, this threshold is exponentially increasing.  So when designing a meaningful pool, this return on investment mindset has to be incorporated.

Happy farming!

**Comment by dorkmo** [comment]
New pooling features coming soon

**Reply by markputrella74** [reply]
I'm 100% agree with you. Everybody is plotting, so the network space is increasing....you do plotting but the average possibility to win is always the same.....but you need more HDD !!! .... I think they should put a limit or similar to the amount of plots generated.....

**Comment by JordyBaylac** [comment]
https://www.chiaexplorer.com/chia-farming-pools
